### ISSUE TITLE
Print Error in Excel generates Java

### ISSUE THREAD

**Issue by muerto7** [WilldooIT/Pentaho-reports-for-OpenERP#108]
Good morning, I was watching the addon, and I had a java error in my reports, I wanted to know if they can evaluate it. 
The error happens when my date parameters, I take a whole month range, or more than 20 days, when I use fewer days, the error does not appear. If required me to spend my Indica prpt report an address to my email (jose.vas.7@gmail.com) 
Thank you.

image error java
http://www.subirimagenes.net/i/140324033220984176.jpg

**Comment by richard-willdooit** [comment]
@muerto7 

I would be interested in seeing your actual prpt file for this.  Converting and comparing dates from OpenERP data sources can get a little tricky, but I am yet to see an error like this.  It is implying that Java is running out of memory, and perhaps there is something recursive being triggered by your report.

Is the report based on an sql query or an OpenERP data source?
Are the date fields being compared to dates or datetimes?
Within the query, are you allowing for context_tz?

Anyway, the prpt file could be of benefit.

**Comment by muerto7** [comment]
@richard-willowit 

I appreciate the help, I give you the link where my prpt file.

https://mega.co.nz/#!dYVkhJSZ!WGA2BZflJlMn_CwoosaxF2ASDCIj6kzGIgbokHtGZu4

**Comment by richard-willdooit** [comment]
I assume you have no problem running the report on longer date ranges through the designer?

If you take this date range selection out, does it work or not?  Maybe it has more to do with the amount of data selected, and my suspicion is more that it may have to do with paging and footers and stuff more than date selections.  You can confirm this by removing the date from the where clause or by hardcoding the comparison to date selections.

Are you definitely running with the latest build?
If you could grab the latest .war file from our website and redeploy it on Tomcat, that may help, as one of the dependencies has been upgraded to PostgreSQL 9, not 8.

Overall, nothing appears to stand out and so I tend towards the paging / footers type problem.

**Comment by muerto7** [comment]
Effectively, work in PRD its ok, I have only problems with OpenERP. 

I did the tests and removing paging and dates footer as you mentioned, did not work. 

The report answers me with pdf format, including pagination and dates (as the report that I spent him). Why not in Excel format?

I am working with the addon Pentaho Open For March 22 this year, I will test the version of April 8 and I will notify my changes. 

I am also working with version 9.3 of postgres
